.github/workflows/stale-issues-pr.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Close stale issues and PRs'
+on:
```

Comment:
Do we need to add the following permissions?
```
permissions:
  contents: write # only for delete-branch option
  issues: write
  pull-requests: write
```

Reply:
Yea we actually should add them but we should not add contents: write that is not needed